maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -186,6 +190,13 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 					}
 					platformMode = false;
 					break;
+				case O3_REFERENCE_APPLICATION:
+					distroProperties = DistroHelper.getDistroPropertiesFromFile(new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "openmrs-distro-3.0.properties"));
+					File source = new File(SDKConstants.MAVEN_PLUGINS_RESOURCE_PATH + "configuration");
+					FileUtils.copyDirectoryToDirectory(source, server.getServerDirectory());
```

Comment:
I think it is considered a good practice to perform `File` interactions within a try block.  By encapsulating file interactions within a try block, you can catch exceptions and handle them appropriately.